pegasus/helpers/hoc_helpers.rb

```diff
@@ -2,15 +2,10 @@
 require 'cdo/graphics/certificate_image'
 
 def create_session_row(row)
```

Comment:
to test, hit the tracking pixel at http://code.org/api/hour/begin_codeorg.png (sub localhost/staging/etc)